### ISSUE TITLE
Reply quote is present when editing a message

### ISSUE THREAD

**Issue by phorcys420** [vector-im/element-android#8612]
### Steps to reproduce

1. DM Chat with someone
2. (Send a message)
3. Reply to a message
4. Edit the reply

#### What did you expect?

I expected that the content that appears in the edit box would only be the reply text.

#### What happened instead?

The edit box also contained the "From" and original text (see video)

### Your phone model

Google Pixel 6a

### Operating system version

Android 13 (TQ3A.230705.001)

### Application version and app store

Element 1.6.5 [40106052] (G-fd6a45a3), Rust SDK 0.6.0 (3fd55c0), Vodozemac 0.4.0 from Play Store

### Homeserver

Synapse 1.85.2

### Will you send logs?

No

### Are you willing to provide a PR?

Yes

**Comment by phorcys420** [comment]
Sorry all, the video wouldn't upload on GH because it's too big.
Here's a link to it:
https://wormhole.app/1QDaN#jchkN_MhvL4GbjiXw5n0Yw

**Comment by Palloxin** [comment]
https://github.com/vector-im/element-android/assets/75091899/a025076c-a1cd-4884-9bc7-a2c243f04def

A video reproduction of the bug.

**Comment by phorcys420** [comment]
I thought it was clear, but I did already include a video in my issue!
In the "What happened?" section.

**Comment by Palloxin** [comment]
It says " not found". not sure if it is only me

**Comment by pludi** [comment]
Sounds like the same issue as in #8602

**Comment by phorcys420** [comment]
> It says "Not found". Not sure if it is only me

Oh yeah, you're right!
Sorry.

> Sounds like the same issue as in https://github.com/vector-im/element-android/issues/8602

I'm unsure given that #8602 contains multiple distinct issues

**Comment by pludi** [comment]
> > Sounds like the same issue as in #8602
> 
> I'm unsure given that #8602 only mentions text displaying issues.

Below the first screenshots there's a section about "duplicate in reply to …:"

**Comment by Luka5W** [comment]
yep. its addressed in my issue and fixed in my pull request. its very annoying  and has been there for a while (#8377) now. no idea how this commit could pass the QA. can't believe that no one has fixed this/ has noticed this behavior.

but features > stable app i guess :clown_face: and its only a major feature to edit replies/ markdown messages so... the label O-Uncommon and S-ToLeRaBlE is completely appropiate here :clown_face: :clown_face: :clown_face:

btw. i would think twice if you put [any effort](https://github.com/matrix-org/matrix-react-sdk/pull/9240#issuecomment-1702915977) in this project... im not sure if i will if my pr isnt accepted without bigger change requests.
